### PR TITLE
Use old financial support field only for 2024 courses and previous cycles only

### PR DESCRIPTION
**Pull request by tomas-stefano**
## Context

Provider raised a support ticket about financial support duplication, Reading the code I understood this field is only for "< 2025 courses"

Support ticket: 

https://trello.com/c/7pXkBeQ8/309-fees-and-financial-support-info-duplicated

### CODE REVIEW

File: app/components/shared/courses/financial_support/fees_and_financial_support_component/view.html.erb [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <% end %>
   <% end %>
 
-  <% if financial_support.present? %>
+  <% if financial_support.present? && !course.recruitment_cycle.after_2024? %>
```
Comment (dcyoung-dev) [inline app/components/shared/courses/financial_support/fees_and_financial_support_component/view.html.erb:22]: It's not possible to view courses from previous cycles - how do you feel about completely removing this from the view?

**Comment by tomas-stefano** [comment]
[Remove duplicated financial support section from Find](https://trello.com/c/fEgmMhGb/2466-remove-duplicated-financial-support-section-from-find)

File: app/components/shared/courses/financial_support/fees_and_financial_support_component/view.html.erb (hunk 1 above)
Comment (tomas-stefano) [reply app/components/shared/courses/financial_support/fees_and_financial_support_component/view.html.erb:22]: I was thinking that, but also I was thinking that providers are worry for tomorrow and I am sign off soon too. I can try give it a go for 10 minutes and see if is only few specs breaks to change.

Comment (tomas-stefano) [reply app/components/shared/courses/financial_support/fees_and_financial_support_component/view.html.erb:22]: Also the financial support is all over the data (on enrichments) and were copied over to this cycle. I think we should amend somehow as well.

Comment (tomas-stefano) [reply app/components/shared/courses/financial_support/fees_and_financial_support_component/view.html.erb]: > It's not possible to view courses from previous cycles - how do you feel about completely removing this from the view?

This is done and pushed.

**Review by dcyoung-dev — APPROVED** [review]

**Pull request merged**